src/Dev/SapphireTest.php

```diff
@@ -334,6 +327,39 @@ protected function setUp(): void
         Email::config()->remove('bcc_all_emails_to');
     }
 
+    /**
+     * Sets the locale which unit-tests should be run in
+     */
+    protected function setI18nLocale(): void
+    {
+        $path = $this->getCurrentRelativePath();
+        $packagistName = '';
+        if (preg_match('#^vendor/([^/]+/[^/]+)/.+#', $path, $matches)) {
+            // Running unit tests of a module in the vendor folder
+            $packagistName = $matches[1];
+        } else {
+            // Running unit tests of a module or project in the root folder
+            $file = BASE_PATH . '/composer.json';
```

Comment:
```suggestion
            $file = Path::join(BASE_PATH, 'composer.json');
```
People might be running a WAMP stack so need windows dir separators

Reply:
Updated